mojo-data/mojos/rulefit/names/v1.00/rulefit_v1.00_names_scores2.txt

```diff
@@ -0,0 +1,29 @@
+[0.0, 0.9669291298494845, 0.03307087015051549]
+[0.0, 0.9669291298494845, 0.03307087015051549]
+[0.0, 0.9669291298494845, 0.03307087015051549]
+[0.0, 0.9669291298494845, 0.03307087015051549]
+[0.0, 0.9669291298494845, 0.03307087015051549]
+[0.0, 0.9669291298494845, 0.03307087015051549]
+[0.0, 0.9669291298494845, 0.03307087015051549]
+[0.0, 0.9669291298494845, 0.03307087015051549]
+java.lang.IllegalArgumentException: categorical value out of range
+java.lang.IllegalArgumentException: categorical value out of range
+java.lang.IllegalArgumentException: categorical value out of range
+java.lang.IllegalArgumentException: categorical value out of range
+java.lang.IllegalArgumentException: categorical value out of range
+java.lang.IllegalArgumentException: categorical value out of range
+java.lang.IllegalArgumentException: categorical value out of range
+java.lang.IllegalArgumentException: categorical value out of range
+java.lang.IllegalArgumentException: categorical value out of range
+java.lang.IllegalArgumentException: categorical value out of range
+java.lang.IllegalArgumentException: categorical value out of range
```

Comment:
L9 to L28: testing scoring outrageously big numbers... -> these categories are unknown -> exception ok

Reply:
L29: this exception is returned when testing predicting on vec too short, the same case as rulefit_v1.00_eyestate_scores2.txt